Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource/helper.go

```diff
@@ -117,7 +117,7 @@ func (m *Helper) Create(namespace string, modify bool, data []byte) (runtime.Obj
 			if err := m.Versioner.SetResourceVersion(obj, ""); err != nil {
 				return nil, err
 			}
-			newData, err := m.Codec.Encode(obj)
+			newData, err := m.RESTClient.GetCodec().Encode(obj)
```

Comment:
> you can pass in a runtime.Object to Request.Body(), and it gets encoded using the RESTClient's codec, right? That seems preferable

For this case in particular, probably, but I'm disinclined to have a second spot where we build the URL unless you feel pretty strongly.

You agree on principle, but want a refactor of the method overall?